app/component/homepage_components/featured_example_card_component.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module HomepageComponents
+  class FeaturedExampleCardComponent < ViewComponent::Base
```

Comment:
We should be able to reuse CircuitCardComponent instead of making a new compoent